### PR TITLE
Add several publications and news

### PR DESCRIPTION
**Pull request by JavierJia**
* Add publications.
* Add talks and news.
* Add contributors.

### CODE REVIEW

File: docs/_posts/2017-08-24-ChenAtARL.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+layout: post
+title: Prof. Li give talks about Cloudberry at Huawei, Google, Teradata, and ARL
```
Comment (chenlica) [inline docs/_posts/2017-08-24-ChenAtARL.md]: Please fix the grammatical error in this sentence.

File: docs/aboutme.md [hunk 2]
```diff
@@ -30,15 +25,22 @@ subtitle:
...
+* [Liangju Chu](https://github.com/liangjuc)
+* [Sicong Liu](https://github.com/lsclovecode)
+* [Vidhyasagar Thirumaraiselvan](https://github.com/vidhya567)
+* [Haohan Zhang](https://github.com/hh1680651)
 
 
 ## Acknowledgement
-This work has been supported by NIH award 1U01HG008488-01, by NSF CNS award 1305430, 
-and by the Army Research Laboratory under Cooperative Agreement Number W911NF-16-2-0110. 
+This work has been supported by NIH award 1U01HG008488-01, by NSF CNS award 1305430,
```
Comment (chenlica) [inline docs/aboutme.md]: Keep the first "by" and remove the other "by"s.

File: docs/pubs.md [hunk 3]
```diff
@@ -6,16 +6,21 @@ subtitle:
 
 ## Papers
 * **Drum: A Rhythmic Approach to Interactive Analytics on Large Data**, Jianfeng Jia, Chen Li, Michael J. Carey.
-[[PDF (Long version)](../img/query-slicing-long.pdf)]
+[[PDF (Long version)](../img/query-slicing-long.pdf)][[Video](https://youtu.be/gztxneqSIDE)]
+* **Caching Geospatial Objects in Web Browsers**, Taewoo Kim, Vidhyasagar Thirumaraiselvan, Jianfeng Jia, Chen Li, ACM SIGSPATIAL 2017
```
Comment (chenlica) [inline docs/pubs.md]: Mention it's a demo paper.

File: docs/_posts/2017-08-24-ChenAtARL.md (hunk 1 above)
Comment (JavierJia) [reply docs/_posts/2017-08-24-ChenAtARL.md]: done.

File: docs/aboutme.md (hunk 2 above)
Comment (JavierJia) [reply docs/aboutme.md]: done

File: docs/pubs.md (hunk 3 above)
Comment (JavierJia) [reply docs/pubs.md]: done

File: docs/pubs.md [hunk 4]
```diff
@@ -7,14 +7,14 @@ subtitle:
 ## Papers
 * **Drum: A Rhythmic Approach to Interactive Analytics on Large Data**, Jianfeng Jia, Chen Li, Michael J. Carey.
 [[PDF (Long version)](../img/query-slicing-long.pdf)][[Video](https://youtu.be/gztxneqSIDE)]
-* **Caching Geospatial Objects in Web Browsers**, Taewoo Kim, Vidhyasagar Thirumaraiselvan, Jianfeng Jia, Chen Li, ACM SIGSPATIAL 2017
+* **Caching Geospatial Objects in Web Browsers (Demo Paper)**, Taewoo Kim, Vidhyasagar Thirumaraiselvan, Jianfeng Jia, Chen Li, ACM SIGSPATIAL 2017
```
Comment (chenlica) [inline docs/pubs.md]: "Demo Paper" is not part of the title, I think.  If so, move it to the end of the line.

File: docs/pubs.md [hunk 5]
```diff
@@ -7,14 +7,14 @@ subtitle:
...
 * **Visual Analytics Ecology for Complex System Testing**,
 Simon Su, Michael Barton, Michael An, Vincent Perry, Chen Li, Jianfeng Jia, Brian Panneton, Visualization in Practice 2017 at IEEE VIS 2017.
 * **Twitter Coverage of Climate Change and Health before and after the 2016 US Presidential Election**, Suellen Hopfer, Miryha Runnerstrom, Jianfeng Jia, Taewoo Kim, Chen Li, American Public Health Association 2017.
 * **Supporting Visualizations on Large Twitter Datasets Using Cloudberry**,
 Shengjie Xu.
 [[PDF](../img/Supporting-Visualizations-on-Large-Twitter-Datasets-Using-Cloudberry.pdf)]
-* **Towards Interactive Analytics and Visualization on One Billion Tweets**,
+* **Towards Interactive Analytics and Visualization on One Billion Tweets (Demo Paper)**,
```
Comment (chenlica) [inline docs/pubs.md]: Ditto.

File: docs/pubs.md (hunk 4 above)
Comment (JavierJia) [reply docs/pubs.md]: done

File: docs/pubs.md (hunk 5 above)
Comment (JavierJia) [reply docs/pubs.md]: done

File: docs/_posts/2017-09-01-JianfengWonGoogleAward.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: Jianfeng Won the Google Graduate Student Award in ICS
+---
+
+Our Ph.D. student, Jianfeng, won the Google Graduate Student Award in ICS. Congratulations!
```
Comment (chenlica) [inline docs/_posts/2017-09-01-JianfengWonGoogleAward.md]: Change "the" to "a".

**Comment by chenlica** [comment]
I gave one more minor comment.  After taking care of it, please do the merge and publishing. Thanks.

**Comment by JavierJia** [comment]
Done.

**Pull request merged**